### PR TITLE
PR for pre-1.0 release 

### PR DESCRIPTION
**Pull request by jd-lara**
This PR is meant to review the current branch for a pre 1.0 release.

### CODE REVIEW

File: UC_test/ramp_test.json [hunk 1]
```diff
@@ -1 +1 @@
...
...":"[ThermalStandard(nothing)]","comment":"devices from which the product can be procured"},{"name":"timeframe","data_type":"Float64","null_value":"0.0","validation_action":"error","comment":"the relative saturation timeframe","valid_range":{"max":null,"min":0.0}},{"name":"requirement","data_type":"Float64","null_value":"0.0","comment":"the required quantity of the product should be scaled by a Forecast"},{"name":"internal","data_type":"InfrastructureSystemsInternal"}],"docstring":"Data Structure for the procurement products for system simulations.","struct_name":"StaticReserve","supertype":"Reserve"},{"fields":[{"name":"name","data_type":"String","null_value":"init"},{"name":"contributingdevices","data_type":"Vector{Device}","null_value":"[ThermalStandard(nothing)]"},{"name":"timeframe","data_type":"Float64","null_value":"0.0","validation_action":"error","valid_range":{"max":null,"min":0.0}},{"name":"requirement","data_type":"TimeSeries.TimeArray","null_value":"[]"},{"name":"internal","data_type":"InfrastructureSystemsInternal"}],"struct_name":"Transfer","supertype":"Service"}]},"basepower":100.0,"runchecks":true,"internal":{"uuid":{"value":"ccf147bd-3373-4c73-bb5a-87812ddb8616"}}}
```
Comment (daniel-thom) [inline UC_test/ramp_test.json:1]: I don't know how these are used but they won't work any more.

**Review by daniel-thom — COMMENTED** [review]
This looks fine. Just need to either update the serialized JSON files or delete them.

File: UC_test/ramp_test.json (hunk 1 above)
Comment (jd-lara) [reply UC_test/ramp_test.json:1]: These are old and can be removed

**Review by daniel-thom — APPROVED** [review]

**Comment by jd-lara** [comment]
Will merge once all other pre-10 branches are approved too.

**Comment by jd-lara** [comment]
I am merging this PR to make the tests pass by default in the packages

**Pull request merged**